conf/script/build-ha-update

```diff
@@ -102,7 +102,13 @@ if [[ -f $consul_kv_dump ]]; then
 fi
 $consul_bin/consul kv export > $consul_kv_dump
 
+# Move cluster into standby mode before deleting the resources
+sudo pcs cluster standby --all
```

Comment:
Is it possible to put a check here if the cluster is already in standby then do not call standby command?

Reply:
Yes, Added check to skip command if the cluster is already in standby mode